services/static-webserver/client/source/class/osparc/component/widget/NodeTreeItem.js

```diff
@@ -300,6 +300,7 @@ qx.Class.define("osparc.component.widget.NodeTreeItem", {
       // The label
       this.addLabel();
       const label = this.getChildControl("label");
+      osparc.utils.Utils.setIdToWidget(label, "nodeTreeItem");
```

Comment:
So this rather labels the text of the node?

Reply:
kenau

Reply:
Ah wait now I bothered to read the mr text